config.yml

```
@@ -25,6 +25,11 @@ Metrics/AbcSize:
 
 Metrics/MethodLength:
   Max: 15
+  CountAsOne: [hash, array, method_call]
+
+RSpec/ExampleLength:
+  Max: 15
+  CountAsOne: [hash, array, method_call]
```

Agreguemos los heredoc aquí también.

```suggestion
  CountAsOne: [array, hash, heredoc, method_call]
```